server/api/core/email.js

```diff
@@ -0,0 +1,59 @@
+import { Meteor } from "meteor/meteor";
+import { check } from "meteor/check";
+import { Job } from "meteor/vsivsi:job-collection";
+import { Jobs, Packages, Templates } from "/lib/collections";
```

Comment:
Yes, the `Assets` import is missing, but that's not where it comes from.  It's a Meteor global and...

> Currently, it is not possible to import `Assets` as an ES6 module. Any of the `Assets` methods below can simply be called directly in any Meteor server code.

https://docs.meteor.com/api/assets.html